sfh/datasets/mergers/kinetic_reduced.py

```diff
@@ -0,0 +1,98 @@
+"""Mergers_kinetic reduced dataset."""
```

Comment:
This is a good idea, but there is a actually a better way to do this ;-) you can define variants of TensorFlow datasets, here is one example from a package I develop:
https://github.com/CosmoStat/autometacal/blob/974b2711ca9fd840c551fab30b0775ab2e5b0876/autometacal/python/datasets/gal_gen.py#L49

Reply:
but it's more to let you know that this is possible than anything else.

Reply:
Thanks @EiffL, I'll have a look at it and try to modify our dataset accordingly.

Reply:
Commit a9f30bddac1b8fc89f877902b24ed151df3bb3e6 implement variant of the dataset MergersKinetic.